docs/pipeline/security/customer-managed-keys/index.md

```diff
@@ -0,0 +1,260 @@
+
+:::note
+
+This is a bolt-on security feature available for enterprise customers using BDP PMC (Private Managed Cloud) or BDP Cloud deployments.
+
+:::
+
+AWS Custom KMS Keys ensure that all data stored at-rest in AWS services (Kinesis, SQS, and S3) are encrypted with a key that you own and control. This provides:
+
+- Enhanced security: full control over encryption keys
+- Compliance: meet regulatory requirements for data encryption
+- Audit control: ability to revoke access to encrypted data
+- Key management: centralized control over data encryption
+
+## Prerequisites
+
+To use custom KMS keys, you will need:
+- AWS account with appropriate permissions to create and manage KMS keys
+- Access to your AWS Console in the same region as your Snowplow pipeline
+- Snowplow pipeline account ID (provided by your Customer Success Manager)
+- Security team approval for cross-account key sharing
+
+This feature has some limitations:
+- EMR shredder: if you're using the legacy EMR shredder process, it may not fully respect custom KMS keys for all output. This system is being deprecated.
+- Single key recommendation: while it's technically possible to use different keys for different services, we recommend using a single key for all pipeline resources for simplicity.
+- Regional restrictions: keys must be created in the same region as your pipeline infrastructure.
+
+## Step 1: Create your custom KMS key
+
+Go to the AWS Console in the same region as your Snowplow pipeline and navigate to **KMS** → **Customer managed keys**. The URL will look similar to `https://us-east-1.console.aws.amazon.com/kms/home?region=us-east-1#/kms/keys`.
+
+Click **Create key** and configure with the following settings:
+
+|       Setting       |                      Value                       |
+| :-----------------: | :----------------------------------------------: |
+|      Key type       |                    Symmetric                     |
+|      Key usage      |               Encrypt and decrypt                |
+| Key material origin |                       KMS                        |
+|     Regionality     |                  Single region                   |
+|        Alias        | `snowplow-pipeline-key` (or your preferred name) |
+
+When setting key administrators:
+
+- Key administrators: select the roles/users from your security team that need administrative access to the key
+- Other AWS Accounts: add your Snowplow pipeline account ID here
+
+:::note
+The "Other AWS Accounts" step is essential. Without adding the Snowplow pipeline account ID, the encryption will not work.
+:::
+
+## Step 2: Key policy configuration
+
+The key creation process will generate a policy similar to the one below. We strongly recommend allowing root access for the Snowplow pipeline account:
+
+```json
```

Comment:
Maybe this would be better in a collapsible block

Reply:
Added a collapsible block